UKHomeOffice/nmsw-ui
# NMSW-1375 Update UPT (ETA) content

# Ticket



----

## AC
There are content additions going live as per NMSW-1372 to production prior to 2.0 feature release. We need to replicate those changes into the 2.0 branch, and add the changes to the UPT results page.

- Add extra line with the UPT email address to the help page -> merged to 2.0.0 as part of the rebase from main
- Update all content on interstitial page -> merged to 2.0.0 as part of the rebase from main
- Update content on UPT results page as per prototype link : https://nmsw-prototype.dev.nmsw.homeoffice.gov.uk/mvp-3/status-checker-results?passengerdetails=noboard


----

## To test

- make sure you do not have an ETA cookie
- sign in
- > you should see interstitial page with new content
- sign out
- sign in
- > you should not see interstitial page
- test the links
- > they should all open in a new tab
- go to help link
- > you should see the line with the new email address
- go to a declaration UPT Results page
- > you should see the new content

----

## Notes
